ethcore/trace/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+description = "Transaction tracing"
+name = "trace"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+common-types = { path = "../types"}
```

Comment:
Can we move here all trace types that this module uses from `common-types` crate?

Reply:
Not sure I understand what you mean. The only type from `common-types` that is used in the `trace` crate is `BlockNumber`:

```
$ rg -F "use common_types" ethcore/trace/
ethcore/trace/src/lib.rs
19:use common_types::BlockNumber;

ethcore/trace/src/db.rs
29:use common_types::BlockNumber;
364:	use common_types::BlockNumber;

ethcore/trace/src/import.rs
19:use common_types::BlockNumber;

ethcore/trace/src/types/localized.rs
21:use common_types::BlockNumber;
```

Reply:
or did you mean moving types from `trace` to `common-types` maybe?

Reply:
Ah, I forgot that `BlockNumber` type is defined in `common_types`

> or did you mean moving types from trace to common-types maybe? 

The other way round, I was thinking of moving `struct Filter` which is located in `ethcore/types/src/trace_filter.rs`, but I just double checked and I can see now, that it is actually only used by client. So please ignore my previous comment 🙃

Reply:
> Ah, I forgot that `BlockNumber` type is defined in `common_types`

Btw I noticed we define __two__ `BlockNumber` types, the other is [here](https://github.com/paritytech/parity-ethereum/blob/7c335e87642b848f61ab3ee7bbff5964b53d6f11/rpc/src/v1/helpers/poll_filter.rs#L28).

Reply:
It's a typedef anyway, so it really doesn't matter that much, since you can just use `u64` (which I believe we do in many places) and won't even notice that you don't use "the correct" type.